AsyncDisplayKit/Layout/ASStaticLayoutSpecDimension.h

```diff
@@ -0,0 +1,73 @@
+/*
+ *  Copyright (c) 2015-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <UIKit/UIKit.h>
+#import <AsyncDisplayKit/ASBaseDefines.h>
+#import <AsyncDisplayKit/ASDimension.h>
+
+/** 
+ Expresses a size with relative dimensions.
+ Used by ASStaticLayoutSpec.
+ */
+typedef struct {
+  ASRelativeDimension width;
+  ASRelativeDimension height;
+} ASRelativeSize;
```

Comment:
I created this separated header for those size types because they are only used in static layout spec. So if one doesn't use the spec, one won't need to bother!